code/__DEFINES/construction.dm

```diff
@@ -109,9 +101,26 @@
 #define CAT_SPAGHETTI "Spaghettis"
 #define CAT_ICE "Frozen"
 #define CAT_DRINK "Drinks"
+
+//crafting defines
+#define CAT_WEAPON_RANGED "Weapons Ranged"
+#define CAT_WEAPON_MELE "Weapons Melee"
```

Comment:
```suggestion
#define CAT_WEAPON_MELEE "Weapons Melee"
```

Reply:
& places where it's used

Reply:
> I noticed that the recipes for the rake, wooden bucket, durathread clothes and fermentation barrel were removed. Was that intentional?

Yes. The crafting menu is now populated with stack recipes and they are duplicates.